libr/bin/p/bin_elf.inc

```diff
@@ -209,6 +210,7 @@ static RList* sections(RBinFile *bf) {
 				ptr->perm |= R_PERM_R;
 			}
 			r_list_append (ret, ptr);
+			bf->o->num_pure_sections++;
```

Comment:
this can be bool. and will be good to make this change for all the bin formats, not just in elf.

Reply:
ok bool in f8847b6635e1637f149996196e788249bb976d89 ... have to start somewhere regarding those bin formats